source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -6909,6 +6909,7 @@
         'returns': 'int32'
     },
     'GetExtendedErrorInfo': {
+        'codegen_method': 'private',
```

Comment:
![image](https://user-images.githubusercontent.com/111916962/193682620-7fcfda13-fc47-4650-b24b-e27e8e481d44.png)

Reply:
I believe this will be a breaking change but I think this next release is expected to be a generally breaking release so now might be the time to add this change if we want to.

This does remove the stubs from the proto which I believe is what seems like the appropriate behavior based on the bug referenced. However I recommended Illya push this change so we can have the conversation of whether or not we want to push this change. I do think we should run this by @harsha-bhushan when he gets back as well for final confirmation.

Reply:
Now that you've added the updated generated files, this PR makes more sense. And thanks for including the I/O Trace snapshot. I'll mark this issue as resolved.